### PR TITLE
feat: bzz.link support

### PR DESCRIPTION
**Pull request by AuHau**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -53,7 +60,7 @@ export const SUPPORTED_LEVELS = ['critical', 'error', 'warn', 'info', 'verbose',
 export type SupportedLevels = typeof SUPPORTED_LEVELS[number]
 
 export const DEFAULT_BEE_API_URL = 'http://localhost:1633'
-export const DEFAULT_HOST = '127.0.0.1'
+export const DEFAULT_HOST = 'localhost:3000'
```
Comment (vojtechsimetka) [inline src/config.ts:63]: Probably don't want the port here, right?
```suggestion
export const DEFAULT_HOST = 'localhost'
```

Comment (AuHau) [reply src/config.ts]: Well that was because of the confusions between HOST and HOSTNAMEs 😅 But yeah nope.

File: README.md [hunk 2]
```diff
@@ -96,7 +104,7 @@ npm run start
 | BEE_API_URL             | http://localhost:1633       | URL of the Bee node API                                                                                    |
 | BEE_DEBUG_API_URL       | undefined                   | URL of the Bee node Debug API. Only used and required when postage stamps autobuy is enabled.              |
 | AUTH_SECRET             | undefined                   | Authentication secret, disabled if not set (this secret is checked in the request header `authorization`). |
-| HOST                    | 127.0.0.1                   | Hostname of the proxy.                                                                                     |
+| HOST                    | localhost                   | Hostname of the proxy.                                                                                     |
```
Comment (vojtechsimetka) [inline README.md:107]:
```suggestion
| HOSTNAME                | localhost                   | Hostname of the proxy.                                                                                     |
```

File: src/index.ts [hunk 3]
```diff
@@ -28,8 +28,8 @@ async function main() {
   }
 
   // Start the Proxy
-  const server = app.listen(port, host, () => {
-    logger.info(`starting gateway-proxy at ${host}:${port}`)
+  const server = app.listen(port, () => {
+    logger.info(`starting gateway-proxy at ${host}`)
   })
```
Comment (vojtechsimetka) [inline src/index.ts:33]: This change should probably be reverted, right?

File: test/config.spec.ts [hunk 4]
```diff
@@ -32,15 +32,15 @@ describe('getAppConfig', () => {
...
     expect(config.port).toEqual(DEFAULT_PORT)
   })
 
   it('should set port and host', () => {
     const HOST = '0.0.0.0'
     const PORT = '5000'
 
-    const config = getServerConfig({ HOST, PORT })
+    const config = getServerConfig({ HOSTNAME: HOST, PORT })
     expect(config.host).toEqual(HOST)
```
Comment (vojtechsimetka) [inline test/config.spec.ts:44]: Lets just rename `HOST` to `HOSTNAME` :)

File: test/config.spec.ts [hunk 5]
```diff
@@ -32,16 +32,16 @@ describe('getAppConfig', () => {
...
   })
 
   it('should set port and host', () => {
-    const HOST = '0.0.0.0'
+    const HOSTNAME = '0.0.0.0'
     const PORT = '5000'
 
-    const config = getServerConfig({ HOST, PORT })
-    expect(config.host).toEqual(HOST)
+    const config = getServerConfig({ HOSTNAME: HOSTNAME, PORT })
```
Comment (vojtechsimetka) [inline test/config.spec.ts:43]:
```suggestion
    const config = getServerConfig({ HOSTNAME, PORT })
```

**Review by vojtechsimetka — APPROVED** [review]
Works great, thanks @AuHau !

**Pull request merged**